components/size_aware_image.jsx

```diff
@@ -361,34 +362,64 @@ export default class SizeAwareImage extends React.PureComponent {
         );
     }
 
-    renderImageOrPlaceholder = () => {
+    renderImageOrFallback = () => {
         const {
             dimensions,
+            fileInfo,
         } = this.props;
 
-        let placeHolder;
+        let ariaLabelImage = localizeMessage('file_attachment.thumbnail', 'file thumbnail');
+        if (fileInfo) {
+            ariaLabelImage += ` ${fileInfo.name}`.toLowerCase();
+        }
+
+        let fallback;
 
         if (this.dimensionsAvailable(dimensions) && !this.state.loaded) {
-            placeHolder = (
-                <div
-                    className={`image-loading__container ${this.props.className}`}
-                    style={{maxWidth: dimensions.width}}
-                >
-                    {this.renderImageLoaderIfNeeded()}
-                    <svg
-                        xmlns='http://www.w3.org/2000/svg'
-                        viewBox={`0 0 ${dimensions.width} ${dimensions.height}`}
-                        style={{maxHeight: dimensions.height, maxWidth: dimensions.width, verticalAlign: 'middle'}}
-                    />
-                </div>
-            );
+            const ratio = dimensions.height > 350 ? 350 / dimensions.height : 1;
```

Comment:
Where does this 350px height come from? Could you move it to a constant so that it has a proper name?

Reply:
It comes from https://github.com/mattermost/mattermost-webapp/blob/10bc28cf4c79fb90b5a05e4429684e6714b1bef6/sass/components/_files.scss#L208 I don't know a css way of doing that and keeping image ratio.

Sure, I should have did it already ! Thanks a lot for the review